### ISSUE TITLE
[1.12.2] No red environment

### ISSUE THREAD

**Issue by JonasCiss** [lumien231/Bloodmoon#24]
If Optfine C 7 is present the sky is red but the environment is normal. If Optifine is not present everything works as fine as it should.

**Comment by JonasCiss** [comment]
Forge: 1.12.2 - 14.23.1.2566
The sky and the moon is red, but not the blocks. It is only present if Optifine is installed.
![2018-01-05_21 45 58](https://user-images.githubusercontent.com/26455012/34650021-0e149e20-f3ba-11e7-9fe9-17d7fcc0d4cc.png)

**Comment by lumien231** [comment]
Yeah, i don't think i can fix this due to the way Optifine is handling the lightmap now. Maybe try to play around with the Optifine Settings a bit, see if something makes it work.

**Comment by JonasCiss** [comment]
I wasn't unable to find way to get it working.

**Comment by sp614x** [comment]
If a resource pack is loaded with custom lightmap then it would overwrite the vanilla lightmap.
If Bloodmoon is patching EntityRenderer.updateLightmap() then the patch may be failing when OptiFine is loaded.
If Bloodmoon is using WorldProvider.getLightmapColors() then OptiFine should be complaining in the error log that something is wrong.

**Comment by SSKirillSS** [comment]
No information on this issue has appeared?
